docs/_quarto.yml

```diff
@@ -127,7 +127,117 @@ format:
     css: styles.css
     toc: true
 
-    # TODO: remove this after proper theme settled and ready to merge
-    # if using the vapor theme, adjust the text color optionally
-    #fontcolor: fuchsia
-    #linkcolor: fuchsia
+quartodoc:
+  package: ibis
+  title: Reference
+  sidebar: reference/_sidebar.yml
+  sections:
+    - title: Expressions
+      desc: ""
+      package: ibis.expr.types
+      contents:
+        - kind: page
+          package: ibis
+          path: top_level
+          summary:
+            name: Top-level APIs
+            desc: These methods and objecst are available directly on the `ibis` module.
+          contents:
+            - name: and_
+              dynamic: true
+
+        - kind: page
+          path: expression-tables
+          package: ibis.expr.types.relations
+          summary:
+            name: Table Expressions
+            desc: Table expressions form the basis for most Ibis expressions.
+          contents:
+            - Table
+            - GroupedTable
+
```

Comment:
+1. Definitely would be nice to "inline" this information similar to how `mkdocstrings` does it.

For example having a file `relations.qmd` with this content

```markdown
::: ibis.expr.types.relations
```

or something similar.

Reply:
Ah, it's currently not possible for quartodoc to generate API docs by manually writing qmds. The challenge is that we need to create the sphinx inventory file during `quartodoc build`, prior to rendering qmds.

The other piece is that the yaml config is used to generate the reference page (e.g. https://machow.github.io/quartodoc/api/). 

In case it feels a bit more manageable--the shiny team ended up...

*  putting all the quartodoc config in a separate yaml file ([`_quartodoc.yml`](https://github.com/posit-dev/py-shiny/blob/main/docs/_quartodoc.yml))
* running `quartodoc build --config _quartodoc.yml --verbose` to generate their API qmd files

Reply:
The big architectural challenge is that most documentation tools (sphinx, mkdocs) allow some interaction with python extensions, but quarto uses lua filters. So we can't really tap quartodoc renderers etc.. from inside quarto.

There's are a thing called [notebook filters](https://quarto.org/docs/extensions/nbfilter.html) that let you pre-process an ipynb from python, and can be used for arbitrary python finagling. (This is used a lot by the [nbdev site](https://nbdev.fast.ai/))